system/logging/appenders/ConsoleAppender.cfc

```diff
@@ -221,4 +210,21 @@ component accessors="true" extends="coldbox.system.logging.AbstractAppender"{
 		return this;
 	}
 
+
```

Comment:
Since we have moved this to a UDF and most likely tons of appenders can leverage it, I would suggest you send this to the `AbstractAppender` instead so any appender can leverage the locking utility.

Reply:
Will do!